admin/sections/config/chat.tsx

```diff
@@ -46,7 +45,20 @@ export const ChatConfig = ({
         render={({ field }) => (
           <FormItem className="pt-4">
             <FormLabel>Conversation questions</FormLabel>
-            <FormControl>
+            <FormControl
+              onBlur={() => {
+                submit(form.getValues(), false);
+              }}
+              onChange={() => {
+                // Handle for the case when the user deletes a question
+                if (
+                  form.getValues().conversation_starters.length <
+                  field.value?.length
+                ) {
+                  submit(form.getValues(), false);
+                }
+              }}
+            >
               <MultiInput {...field} />
```

Comment:
can't we just add `onDelete` to `MultiInput` instead?
```suggestion
            >
              <MultiInput {...field} onDelete={() => {
                submit(form.getValues(), false);
              } />
```

Reply:
Great idea, i just added it. Thanks